knimin/tests/test_ag_edit_barcode.py

```diff
@@ -0,0 +1,108 @@
+from unittest import main
+
+from tornado.escape import url_escape
+
+from knimin.tests.tornado_test_base import TestHandlerBase
+from knimin import db
+
+
+class TestAGEditBarcodeHandler(TestHandlerBase):
+    def test_get_not_authed(self):
+        response = self.get('/ag_edit_barcode/')
+        self.assertEqual(response.code, 200)
```

Comment:
I thought there was an issue about this but I'm not seeing it right now -- should unauthorized attempts result in a `200` or `403`?